.appveyor.yml

```diff
@@ -17,8 +17,7 @@ install:
   - set PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%
   # The package installation via our setup.py file uses easy_install, which fails to correctly install lxml.
   # As some Python environments don't have lxml preinstalled, install it here to avoid errors during the execution in those cases.
-  - python -m pip install lxml
-  - python -m pip install --user .
+  - python -m pip install --user ".[dev]"
 
 test_script:
-  - python setup.py nosetests --test benchexec.tablegenerator
+  - python -m nose --tests benchexec.tablegenerator
```

Comment:
Just to confirm your proposal: This is a better way to run the tests. :+1: 

Not only is it required for getting rid of `setup.py` eventually, the [test output](https://ci.appveyor.com/project/sosy-lab/benchexec/builds/46428757/job/0pvyg2nm59g37xb3#L40) is also not cluttered anymore with installation stuff as [before](https://ci.appveyor.com/project/sosy-lab/benchexec/builds/46351006/job/r035xgrtp3q2i7yf#L40).

Reply:
I did not compare the test output with a previous build, nice!